packages/web-react/src/constants/dictionaries.ts

```diff
@@ -1,4 +1,11 @@
 /* Alignment */
+export const Alignment = {
```

Comment:
This is not in our dictionary. 

Either remove it from here and make it local for the component or add it to the notion and global dictionaries docs in the repo root, but we should be all aware of that (including Michal).

Reply:
fixed